### PR TITLE
Swagger doc improvements

### PR DESCRIPTION
**Pull request by BjarkeMeier**
## Description
Update documentation from this:

image

to this:

image

## References

* #782

### CODE REVIEW

File: source/dotnet/Services/WebApi/Startup.cs [hunk 1]
```diff
@@ -41,24 +41,25 @@ public void ConfigureServices(IServiceCollection services)
...
 
         services.AddApiVersioning(config =>
         {
-            config.DefaultApiVersion = new ApiVersion(1, 0);
+            config.DefaultApiVersion = new ApiVersion(3, 0);
             config.AssumeDefaultVersionWhenUnspecified = true;
             config.ReportApiVersions = true;
         });
 
+        // Add ApiExplorer to discover versions
```
Comment (lasrinnil) [inline source/dotnet/Services/WebApi/Startup.cs:55]: IMO `AddVersionedApiExplorer()` is pretty well named, consider removing the comment.

File: source/dotnet/Services/WebApi/Startup.cs [hunk 2]
```diff
@@ -77,7 +78,17 @@ public void Configure(IApplicationBuilder app)
         }
 
         app.UseSwagger();
-        app.UseSwaggerUI();
+
+        var apiVersionDescriptionProvider = app.ApplicationServices.GetRequiredService<IApiVersionDescriptionProvider>();
+        app.UseSwaggerUI(options =>
+        {
+            foreach (var description in apiVersionDescriptionProvider.ApiVersionDescriptions.Reverse())
```
Comment (lasrinnil) [inline source/dotnet/Services/WebApi/Startup.cs:85]: What does/works the Reverse() in this case?

File: source/dotnet/Services/WebApi/WebApi.csproj [hunk 3]
```diff
@@ -44,7 +45,7 @@ limitations under the License.
 
   <ItemGroup>
     <Compile Include="..\..\Contracts\BatchDtoV2.cs">
-      <Link>Controllers\V2\BatchDtoV2.cs</Link>
+      <Link>V2\BatchDtoV2.cs</Link>
```
Comment (lasrinnil) [inline source/dotnet/Services/WebApi/WebApi.csproj:48]: intended?

**Review by lasrinnil — DISMISSED** [review]

File: source/dotnet/Services/WebApi/Startup.cs [hunk 4]
```diff
@@ -41,24 +41,25 @@ public void ConfigureServices(IServiceCollection services)
...
+            config.DefaultApiVersion = new ApiVersion(3, 0);
             config.AssumeDefaultVersionWhenUnspecified = true;
             config.ReportApiVersions = true;
         });
 
+        // Add ApiExplorer to discover versions
+        services.AddVersionedApiExplorer(setup =>
+        {
+            setup.GroupNameFormat = "'v'VVV";
+            setup.SubstituteApiVersionInUrl = true; // TODO: Remove?
```
[the remark this answers is not in the document]
Comment (BjarkeMeier) [reply source/dotnet/Services/WebApi/Startup.cs]: Yes, I've removed the comment 😉 
Substitution is needed in the V2 API.

File: source/dotnet/Services/WebApi/Startup.cs (hunk 1 above)
Comment (BjarkeMeier) [reply source/dotnet/Services/WebApi/Startup.cs]: Ok. Comment removed.

File: source/dotnet/Services/WebApi/Startup.cs (hunk 2 above)
Comment (BjarkeMeier) [reply source/dotnet/Services/WebApi/Startup.cs:85]: Doc added.

**Pull request merged**